tools/metrics/import/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.8'
+
+networks:
+  monitoring:
+    driver: bridge
+
+services:
+  init_prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus_prestart
+    user: root
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        chown -R 65534:65534 /prometheus
+    volumes:
+      - ./data/prometheus/:/prometheus/
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
```

Comment:
I would have this one parametrizable and/or suffixed so we could more easily compare two post-mortem datasets.

Reply:
You'd need to have different grafana deployments too. I think it would be of interest to discuss this with Quentin since these "docker compose" operations and files will be generated by the CLI down in the future.

Reply:
Discussed with Quentin and the docker-compose will be removed and replaced with a Terraform deployment with all three imported resources.